### PR TITLE
Update principles for compensation philosophy

### PR DESCRIPTION
**Pull request by choldgraf**
In a recent meeting with @GeorgianaElena and @damianavila , we realized that 2i2c's policy about "negotiation for salary within a role" was not clear. We hinted that we do not negotiate but we didn't explicitly state it. This PR makes this more explicit, and generally tries to clarify and clean up our salary rationale and policy.

I don't believe that it **changes** our policy, merely clarifies it, so I don't think this requires review from our steering council.

- related to: https://github.com/2i2c-org/meta/issues/171

### CODE REVIEW

File: people/compensation.md [hunk 1]
```diff
@@ -2,17 +2,32 @@
...
+- **Fixed salaries for a role**. We do not negotiate salaries within a given role.
+  Salaries are fixed for a given role and tenure[^tidelift].
+- **Equal pay for equal work**. We practice [equal pay for equal work](https://en.wikipedia.org/wiki/Equal_pay_for_equal_work) across our team. Salaries do not distinguish between geographic location or cost of living.
+
+[^tidelift]: This is because we believe that allowing negotiation introduces bias that benefits individuals who are more confident, aggressive, or experienced. This also correlates heavily with demographics and personal qualities that are not tied to the ability to do the job well. Instead we try to be as transparent as possible for the rationales at a given level. See [the Tidelift blog post on their no-negotiation policy for salaries](https://blog.tidelift.com/why-we-have-a-no-negotiation-policy-at-tidelift) for one rationale for this practice, and [this blog post for an argument *against* negotiation bans](https://www.shrm.org/hr-today/news/hr-magazine/Pages/0915-salary-negotiation-bans.aspx).
```
Comment (GeorgianaElena) [inline people/compensation.md:27]: :heart:

**Review by GeorgianaElena — APPROVED** [review]
Thank you  @choldgraf!

File: people/compensation.md [hunk 2]
```diff
@@ -2,17 +2,32 @@
...
+We are a young organization, and this page describes the salary policy that we aim to enact in the coming years.
+It may not accurately reflect our current practices, but we share it for transparency and to provide a goal for our team to shoot for.
+:::
+
+Our salary policy should ensure that our team feels financially supported and respected in their roles, and should be aligned with our values and mission as a non-profit organization.
+Team members should feel that 2i2c is a competitive workplace from a salary perspective, while recognizing that we do not have the financial resources of most tech companies.
+Below are a few guiding principles of our salary policy:
+
+- **Benchmarked to non-profit tech**. We should benchmark our salaries based on similar roles in our sector (non-profits that do technical or research work, depending on the role). See [](compensation:salaries:guidestar) for the current process we follow.
+  - If we determine that a sector is systemically under-paying a given role in a way that makes us uncompetitive, then we may increase the salary band for the role to be more aligned with the private sector.
```
Comment (damianavila) [inline people/compensation.md:20]: Thanks for adding this piece. I do think our "guide" should be something else than the guidestar one for the Eng team, and the upcoming proposal I am working on takes that point into account.

Comment (damianavila) [reply people/compensation.md:20]: I also ack Eng "being different" may cause some conflicts that we will need to address as well so other areas of the organization feel OK. My proposal will include a specific point for that as well.

File: people/compensation.md [hunk 3]
```diff
@@ -39,16 +54,15 @@ Here are a few tips to understand the GuideStar reports:
...
+### List of salary bands
 
-Benefits are offered by {term}`Code for Science and Society`.
-They are offered via [TriNet](https://www.trinet.com/) as described in [the CS&S employee handbook](https://drive.google.com/file/d/1anHo8P09gjGLnUfGj2ceSDxvJTYwMeS1/view?usp=sharing).
+:::{admonition} To be updated
+We are working on updating our salary bands across 2i2c, see [this issue for details](https://github.com/2i2c-org/meta/issues/171).
+:::
 
-## Rationale
+We have [internal salary documents for the specific positions and levels](https://docs.google.com/spreadsheets/d/1FJM5pAbc0EWhu4CpPjlbWTMOsZAnivEd2ZBIZIdwpE8/edit?usp=sharing).
```
Comment (damianavila) [inline people/compensation.md:63]: I think that link is pointing to something else...

**Review by damianavila — APPROVED** [review]

File: people/compensation.md [hunk 4]
```diff
@@ -2,17 +2,32 @@
...
+
+- **Benchmarked to non-profit tech**. We should benchmark our salaries based on similar roles in our sector (non-profits that do technical or research work, depending on the role). See [](compensation:salaries:guidestar) for the current process we follow.
+  - If we determine that a sector is systemically under-paying a given role in a way that makes us uncompetitive, then we may increase the salary band for the role to be more aligned with the private sector.
+- **Explicit rationale**. All of our salaries must be tied to explicit roles, with consistent and transparent responsibilities and expectations for entering and advancing a role.
+- **Transparent compensation by role**. Our salaries, benefits, and the roles that they are attached to should be available to all.
+- **Fixed salaries for a role**. We do not negotiate salaries within a given role.
+  Salaries are fixed for a given role and tenure[^tidelift].
+- **Equal pay for equal work**. We practice [equal pay for equal work](https://en.wikipedia.org/wiki/Equal_pay_for_equal_work) across our team. Salaries do not distinguish between geographic location or cost of living.
```
Comment (yuvipanda) [inline people/compensation.md:25]: It would be useful to state that we do this by benchmarking against a high cost of living area, rather than a low one

Comment (choldgraf) [inline people/compensation.md:25]:
```suggestion
- **Equal pay for equal work**. We practice [equal pay for equal work](https://en.wikipedia.org/wiki/Equal_pay_for_equal_work) across our team. Salaries do not distinguish between geographic location or cost of living and are benchmarked against North American salaries in the non-profit tech sector.
```

Comment (choldgraf) [reply people/compensation.md:25]: I added a note that I think reflects what we're currently doing. We are using GuideStar as a starting point, which reflects North American non-profit organizations. There is probably a general skew towards higher-income cities for organizations of our size but it is not necessarily directly related to high-income cities.

**Comment by choldgraf** [comment]
I'm gonna merge this in since we have 2 approvals and I think I've addressed @yuvipanda's suggestion above. I am happy to keep iterating in another PR if you prefer we tweak the language further!

**Pull request merged**